apps/studio/src/lib/db/clients/firebird.ts

```diff
@@ -366,6 +366,7 @@ export class FirebirdClient extends BasicDatabaseClient<FirebirdResult> {
           defaultValue,
           nullable,
           primaryKey,
+          hasDefault: Boolean(defaultValue),
```

Comment:
This won't work if the defaultValue is 0. You should use `!_.isNil`

Reply:
![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/279769/18cc1e4e-b060-4193-a5c1-50c1290762e8)

Reply:
![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/279769/5ec0a54b-c37a-41c5-b4a6-0cd9380baf4d)

Reply:
Didn't think about those being valid default values. Good catch!